sdk/timeseriesinsights/tests.yml

```diff
@@ -1,8 +1,8 @@
 trigger: none
 
 extends:
-  template: ../../eng/pipelines/templates/jobs/archetype-sdk-tests.yml
+  template: ../../eng/pipelines/templates/stages/archetype-sdk-tests.yml
   parameters:
     ServiceDirectory: timeseriesinsights
     Location: westus2
-    SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources-preview)
\ No newline at end of file
+    Clouds: Preview
```

Comment:
Why does this say preview? Should we have updated this to public cloud or something once the service went GA?

@pakrym can you advise?

Reply:
I would leave the `Clouds` out entirely and use the default settings.

If you have a cloud where the service is not supported you can use the `UnsupportedClouds` to express that.

+ @benbp